### ISSUE TITLE
Shell interface

### ISSUE THREAD

**Issue by iefserge** [ArielAbreu/runtime-shell#1]
Hey, great idea to move this into a module! Let's discuss shell interface here.

I'm curious what is `rshell.version`?

**Comment by ArielAbreu** [comment]
Oh, right now, rshell.version is just the app's version.

**Comment by iefserge** [comment]
ah, you can read module version like `require('./package.json').version`

**Comment by ArielAbreu** [comment]
Yes, hmm, I tried testing that but got an error saying it couldn't be found, but, I think it's because I used `__dirname`.

**Comment by ArielAbreu** [comment]
Should I move my PR from runtimejs/runtime#60 to runtime-shell?
